Mathlib/Algebra/Group/Subgroup/Pointwise.lean

```diff
@@ -392,6 +392,13 @@ theorem Normal.conjAct {G : Type*} [Group G] {H : Subgroup G} (hH : H.Normal) (g
 theorem smul_normal (g : G) (H : Subgroup G) [h : Normal H] : MulAut.conj g • H = H :=
   h.conjAct g
 
+theorem normalCore_eq_iInf_conjAct {G : Type*} [Group G] (H : Subgroup G) :
+    H.normalCore = ⨅ (g : ConjAct G), g • H := by
```

Comment:
Is there already a `{G : Type*} [Group G]` in scope at this point of the file? If yes, could you please get rid of these in your lemma?